orchestration/hca_manage/manifest.py

```diff
@@ -44,30 +49,33 @@ def _parse_csv(csv_path: str) -> set[str]:
     return paths
 
 
-def parse_and_load_manifest(env: str, csv_path: str, release_tag: str) -> None:
+def parse_and_load_manifest(env: str, csv_path: str, release_tag: str, pipeline_name: str) -> None:
     paths = _parse_csv(csv_path)
     assert len(paths), "At least one import path is required"
 
-    blob_name = f"load_hca/{release_tag}_manifest.csv"
+    blob_name = f"{pipeline_name}/{release_tag}_manifest.csv"
     storage_client = Client()
     bucket: Bucket = storage_client.bucket(bucket_name=ETL_PARTITION_BUCKETS[env])
     blob = Blob(bucket=bucket, name=blob_name)
     if blob.exists():
-        if not query_yes_no(f"Manifest {blob.name} already exists, overwrite?"):
+        if not query_yes_no(f"Manifest {blob.name} already exists for pipeline {pipeline_name}, overwrite?"):
             return
 
     logging.info(f"Uploading manifest [bucket={bucket.name}, name={blob_name}]")
     blob.upload_from_string(data="\n".join(paths))
-    _reload_repository(_get_dagster_client())
 
 
 def _get_dagster_client() -> DagsterGraphQLClient:
-    return DagsterGraphQLClient("localhost", port_number=8080)
+    try:
+        return DagsterGraphQLClient("localhost", port_number=8080)
+    except DagsterGraphQLClientError:
+        logging.error("Could not connect to dagster instance on port 8080, ensure you are port forwarding")
+        sys.exit(1)
 
 
 def _reload_repository(dagster_client: DagsterGraphQLClient) -> None:
-    result = dagster_client.reload_repository_location(REPOSITORY_LOCATION)
-    if result.status != ReloadRepositoryLocationStatus.SUCCESS:
+    result = dagster_client.shutdown_repository_location(REPOSITORY_LOCATION)
```

Comment:
The partitions are not picked up with just a repository reload, we need to fully shut it down and let k8s spawn a new pod.